Software/javascript/README.md

```diff
@@ -2,16 +2,46 @@
 <!-- press ctrl+K then V to open a preview of the MarkDown file  -->
 =======
 >>>>>>> Stashed changes
-# Page Main Title 
-short description of the directory's subject
+# Vanilla JavaScript 
+Here you will find Vanilla JavaScript CheatSheats, guides, articles, etc...
 
-# Table Of Contents
+# 2 websites with all the answers you'll ever need:
 
-* [subject](#subject) 
+* [W3Schools](https://www.w3schools.com/js).
+* [MDN Mozilla](https://developer.mozilla.org/en-US/docs/Web/JavaScript).
 
-* [subject2](#subject2) 
+# E6 New Syntax:
+* [ES6 new syntax VS ES5 old syntax ](http://es6-features.org/#StatementBodies) in this website you can compare easily ECMA Script-6 and ECMA Script-5
+* [ES6 CheatSheet](ES6-cheatSheet.pdf)
+
+# Methods
+* [Array Methods](https://www.shortcutfoo.com/app/dojos/javascript-arrays/cheatsheet)
+* [W3 Array Methods](https://www.w3schools.com/jsref/jsref_obj_array.asp)
+* [MDN Array Methods](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array)
```

Comment:
a bit redundant to include 3 sources that are almost identical, especially when two of them are already included before.
you should choose which one is best, either most informative, most convenient, etc.. or if all have their merits, add a short explanation why they should be included, eg:
* **site one**  
this is good for beginners
* **site two**  
harder to read but has more information

but unless there is really good reason I think one is enough.